support/ppPrint.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#ifdef SPDLOG_ENABLED
+  #include "spdlog/spdlog.h"
+  #include <spdlog/fmt/bundled/printf.h>
+#endif
+
+#include <Kokkos_Core.hpp>
+#include "ppMacros.h"
+
+namespace pumipic {
+
+  #if defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__) || defined(__SYCL_DEVICE_ONLY__)
+    #define ACTIVE_GPU_EXECUTION
+  #endif
+
+  inline FILE* pp_stdout = stdout;
+  inline FILE* pp_stderr = stderr;
+
+  inline void pp_set_stdout(FILE* out) {
```

Comment:
Since this is in the pumipic namespace,  does it need the `pp` prefix?

Reply:
It doesn't, I updated the name.